ansible/group_vars/all/defaults.yaml

```diff
@@ -14,13 +14,13 @@ kubernetes_cni_version: "0.9.1"
 # The project release closely follows the Kubernetes release cycle, and uses a
 # nearly identical version scheme.
 # IMPORTANT When you update crictl_version, also update crictl_sha256.
-crictl_version: "1.25.0"
+crictl_version: "1.26.1"
```

Comment:
Oh right, want to also change the version of `cri-tools` to 1.26.x. The offline packages also need to be rebuilt with a matching version. https://github.com/mesosphere/dkp-release/blob/8fbee2513f22688283c8ea9dfd323cb9e30c155d/os-packages-bundles/Makefile#L183

Its not strictly required, maybe we can separate that in a different PR, looks like the new package is not yet available https://github.com/mesosphere/konvoy-image-builder/actions/runs/4471534089/jobs/7856545772?pr=748#step:9:2579

Reply:
Merging the k8s bump would also unblock https://github.com/mesosphere/konvoy-image-builder/pull/700